doc/htmldoc/synapses/connection_management.rst

```diff
@@ -256,6 +256,107 @@ must be ``True``.
                       'allow_autapses': False, 'make_symmetric': True}
     nest.Connect(A, B, conn_spec_dict)
 
+.. _tripartite_connectivity:
+
+Tripartite connectivity
+-----------------------
+
+NEST supports creating connections of three node populations using the
+:py:func:`.TripartiteConnect` function:
+
+.. code-block:: python
+
+    TripartiteConnect(pre, post, third, conn_spec)
+    TripartiteConnect(pre, post, third, conn_spec, syn_specs)
+
+``pre``, ``post``, and ``third`` are ``NodeCollections``, defining the nodes of
+origin (`sources`) and termination (`targets`) as well as the third
+factor to be included. Details of the connections created depend on
+the connection rule used.
+
+``conn_spec`` must be provided and must specify a tripartite
+connection rule.
+
+``syn_specs`` is a dictionary of the form
+
+.. code-block:: python
+
+    {"primary": <syn_spec>,
+     "third_in": <syn_spec>,
+     "third_out": <syn_spec>}
+
+where the individual ``syn_spec`` elements follow the same rules as
+for the :py:func:`.Connect` function. Any of the three elements can be
+left out, in which case the synapse specification defaults to
+``"static_synapse"``. The ``"primary"`` synapse specification applies
```

Comment:
Here and below: Use single quotation marks for consistency, e.g., 'primary'.

Reply:
This is fixed.